test/integration/features/basic.feature

```diff
@@ -3,30 +3,42 @@ Feature: Basic
   As a user I can perform basic operations of Minishift and OpenShift
 
   Scenario: User can install default add-ons
-    Given Minishift has state "Does Not Exist"
+   Given Minishift has state "Does Not Exist"
     When executing "minishift addons install --defaults" succeeds
     Then stdout should contain
      """
      Default add-ons anyuid, admin-user, xpaas installed
      """
 
   Scenario: User can enable the anyuid add-on
-    Given Minishift has state "Does Not Exist"
+   Given Minishift has state "Does Not Exist"
     When executing "minishift addons enable anyuid" succeeds
     Then stdout should contain
      """
      Addon 'anyuid' enabled
      """
 
+  @minishift
   Scenario: User can list enabled plugins
-    Given Minishift has state "Does Not Exist"
+   Given Minishift has state "Does Not Exist"
     When executing "minishift addons list" succeeds
     Then stdout should contain
      """
      - anyuid         : enabled    P(0)
      - admin-user     : disabled   P(0)
      """
 
+  @cdk
```

Comment:
I am not so sure about this for several reasons. 

* This makes CDK visible in our tests. I thought we said, we would keep CDK specific tests in the downstream repo
* If we have several of these cases, the feature file will become unwieldy 
* It might be confusing for contributors. It is not clear what 'CDK' means from a community perspective. 

Maybe we should discuss the strategy moving forward.